Packs/Jira/Integrations/JiraV2/JiraV2_test.py

```diff
@@ -9,12 +9,48 @@
     'query': 'status=Open'
 }
 
+integration_args_missing_mandatory = {
+    "summary": "test"
+}
+
+integration_args = {
+    "summary": "test",
+    "projectKey": "testKey",
+    "issueTypeId": "1234"
+}
+
 
 @pytest.fixture(autouse=True)
 def set_params(mocker):
     mocker.patch.object(demisto, 'params', return_value=integration_params)
 
 
+def test_create_issue_command_after_fix_mandatory_args_issue(mocker):
+    from JiraV2 import create_issue_command
+    mocker.patch.object(demisto, 'args', return_value=integration_args)
+    user_data = {
+        "self": "https://demistodev.atlassian.net/rest/api/2/user?accountId=1234", "accountId": "1234",
+        "emailAddress": "admin@demistodev.com", "displayName": "test", "active": True,
+        "timeZone": "Asia/Jerusalem", "locale": "en_US", "groups": {"size": 1, "items": []},
+        "applicationRoles": {"size": 1, "items": []}, "expand": "groups,applicationRoles",
+        "projects": [{'id': '1234', 'key': 'testKey', 'name': 'testName'}]
+    }
+    mocker.patch('JiraV2.jira_req', return_value=user_data)
+    mocker.patch.object(demisto, "results")
+    create_issue_command()
+    assert demisto.results.call_count == 1
+
+
+def test_create_issue_command_before_fix_mandatory_args_issue(mocker):
+    mocker.patch.object(demisto, 'args', return_value=integration_args_missing_mandatory)
```

Comment:
A better practice would be to test each of the args separately, to verify that even if one of them is missing an exception would be thrown.